pkg/flavor/kubernetes/nfs.go

```diff
@@ -965,6 +964,9 @@ func (flavor *Flavor) waitForPVCCreation(claimName, nfsNamespace string) error {
 			log.Infof("pvc %s still not bound to pv, current state %s. waiting(try %d)...", claimName, claim.Status.Phase, i+1)
 			time.Sleep(sleepTime)
 			continue
+		} else if !claim.ObjectMeta.DeletionTimestamp.IsZero() {
+			log.Errorf("existing pvc %s is under deletion, return error to retry later...", claimName)
```

Comment:
log this as debug/trace as it's unrelated to the error below.

Reply:
sure, changed it to log.warnf() as its very rare and useful for debugging without trace levels.